ynput/OpenPype
# Nuke: OCIO env var workflow

## Changelog Description
The OCIO environment variable needs to be consistently handled across all platforms. Nuke resolves the custom OCIO config path differently depending on the platform, so we included the ocio config path in the workfile with a partial replacement using an environment variable. Additionally, for Windows sessions, we replaced backward slashes with a TCL expression.

## Testing notes:
1. activate Colorspace management and add config path with root token
2. open Nuke in the project and see the custom ocio config path was set to some strange values  
![image](https://github.com/ynput/OpenPype/assets/40640033/74f235df-025a-429d-a8d6-790d05b71565)
In my case I was setting the path within `{root[resources]}/...`
![image](https://github.com/ynput/OpenPype/assets/40640033/812b2e4a-67d1-4010-9d7e-b11352f3ba78)
If you want to make a new drive letter from some folder in case you want to test multi root - here it how to do it. 
`subs X: C:\path\to\folder`
3. submit publish deadline farm job
4. make sure your last version .zip package is created and Deadline can see it.
5. all should be rendered as expected

## Dependencies:
https://github.com/ynput/OpenPype/pull/5387
https://github.com/ynput/OpenPype/pull/5411
